Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/openshift/origin-cli:4.2.0
+FROM quay.io/openshift/origin-cli:4.8.0
```

Comment:
For the future, once we have some automation around it, would be to set the version to a build arg passed in at build-time.

Reply:
I checked the repo and i saw an updated image tagged as "latest". Shouldn't we use that instead?

Reply:
yes, we should definitely use a more recent one. you can see what versions clair says are clear here: https://quay.io/repository/openshift/origin-cli?tab=tags